### DISCUSSION TITLE
Perseus moves to Framesurge!

### DISCUSSION THREAD

**Discussion by arctic-hen7** [framesurge/perseus#184] [Announcements]
Today, I'm happy to announce the transfer of the Perseus repository into the *Framesurge* GitHub organization, a new place for developing the frameworks of the future.

For clarity, Perseus' maintainership will remain with me, @arctic-hen7, and nothing will change about the project except for its location on the web. GitHub will automatically transfer all issues, pull requests, stars, contributors, etc. to the new organization, and I have set up a redirect system for the documentation to make sure any existing links still work. Further, Perseus is in no way disconnecting from Sycamore, rather it remains as important as ever.

However, some of you may recall a discussion several months ago on the Sycamore Discord server, in which the idea of merging Perseus into the Sycamore GitHub organization was floated. The reason I've decided not to do this is so that Perseus can be grouped with future projects that are intimately related with it, of which I have a few in the works.

For now, the only other repository in the new organization is for managing Perseus' new official Docker images, which will be maintained by @allen-woods (whom I must thank for the exceptional work he's put into this over the last few months)! For now, this will probably be the only other repository in there, though there are plans for a simpler version of Perseus that can be used with a more generic templating system, which would be compatible with other programming languages, as well as extensions of Perseus into the backend. The possibility of future database integration in some shape is also being considered.

So, in summary, the Perseus repository is being transferred to the new *Framesurge* organization, which will maintain the same governance structures as Perseus, but will provide a place for grouping future projects that extend on Perseus' capabilities, which have been very well received so far in the Rust community. I look forward to continuing to work on this project and future extensions thereof! And, as a final note, thank you to all those who have found, used, and starred Perseus over the last year, and especially to those who have contributed by submitting issues, PRs, and pointing out bugs elsewhere --- it means the world to me to have had such a vibrant community spring up around this project!

**Comment by erlend-sh** [comment]
Congrats! 🎈 

What’s the meaning of the name Framesurge? When I search for it I just get pictures of bike parts 😁 Seems helpfully unique though, since the Perseus repo is already the top result.

**Reply by arctic-hen7** [reply]
Framework + a nifty word that implies progress and electrification, pretty much! The uniqueness was also a key aspect of the choice. The focus will be on building frameworks that make things easier, to enable developers to more rapidly work with powerful, but complex, technologies.
